game.js

```diff
@@ -0,0 +1,80 @@
+let playerPoints = 0;
+let computerPoints = 0;
+let choices = ["Rock", "Paper", "Scissors"];
```

Comment:
This should be `const` since it will not chance in the entire scope

Reply:
Okay.Done